third_party/sycl/sycl_configure.bzl

```diff
@@ -175,8 +176,25 @@ def _create_dummy_repository(repository_ctx):
   repository_ctx.file("crosstool/BUILD", _DUMMY_CROSSTOOL_BUILD_FILE)
 
 
-def _sycl_autoconf_imp(repository_ctx):
+def _sycl_autoconf_impl(repository_ctx):
   """Implementation of the sycl_autoconf rule."""
+  # ARM toolchain bits
+  if "CROSSTOOL_PYTHON_INCLUDE_PATH" in repository_ctx.os.environ:
+    python_include_path = repository_ctx.os.environ["CROSSTOOL_PYTHON_INCLUDE_PATH"]
+  else:
+    python_include_path = "/usr/include/python2.7"
+
+  if _crosscompile(repository_ctx):
+    gcc_toolchain_path = repository_ctx.os.environ["CROSS_TOOLCHAIN"]
+    gcc_toolchain_name = repository_ctx.os.environ["CROSS_TOOLCHAIN_NAME"]
+    opencl_includes = repository_ctx.os.environ["OPENCL_INCLUDES"]
+  else:
+    gcc_toolchain_path = ""
+    gcc_toolchain_name = ""
+    opencl_includes = ""
+  spir_type = repository_ctx.os.environ["BITCODE_TARGET"]
```

Comment:
These are all environment variables. I appreciate that's not perfect but for now it's an easy and configurable way to get this stuff working. Ideally, it would be configure-time, or more fixed, but this is early days.